sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_training_client.py

```diff
@@ -399,7 +400,10 @@ def begin_create_composed_model(
         **kwargs
     ):
         # type: (List[str], Any) -> LROPoller[CustomFormModel]
-        """Creates a composed model from a collection of existing trained models with labels.
+        """Creates a composed model from a collection of existing models that were trained with labels.
```

Comment:
I think you might need an empty line between the one sentence description, and the more in-depth description